ClientSideEncryptionToCPKN/ClientSideKeyVaultKeyToCustomerManagedKey/ExampleSetup/ExampleSetup.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.1.1" />
+    <PackageReference Include="Azure.Security.KeyVault.Keys" Version="4.0.4" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.5.0-preview.1" />
```

Comment:
afaik latest crypto preview was "preview.5" could you bump this? (there might be some slight interface changes)